frontend/src/components/core/Sidebar/styled-components.ts

```diff
@@ -84,6 +84,36 @@ export const StyledSidebarContent = styled.div<StyledSidebarContentProps>(
       zIndex: theme.zIndices.sidebarMobile,
     },
 
+    "& h1": {
+      fontSize: theme.fontSizes.xl,
+      fontWeight: 600,
+    },
+
+    "& h2": {
+      fontSize: theme.fontSizes.lg,
+      fontWeight: 600,
+    },
+
+    "& h3": {
+      fontSize: "1.125rem",
```

Comment:
nit: we may want to have a constant defined in `theme.fontSizes` for this (I'm not entirely sure about this, though, given that there isn't really a natural size between `lg` and `md`)